pyproject.toml

```diff
@@ -16,12 +16,10 @@ classifiers = [
   "Topic :: Security",
 ]
 dependencies = [
-  "httpx==0.28.1",
-  "pydantic==2.12.5",
+  "httpx>=0.28.1",
+  "pydantic>=2.12.5",
   "Authlib>=1.4.1,<2.0.0",
-  "importlib-resources>=6.0.0,<7",
   "click>=8.1.3,<9",
-  "junit-xml>=1.9,<2",
```

Comment:
junit-xml is still in use: https://github.com/onekey-sec/python-client/blob/836fd0e735a7d272ee4b97969776df41578e5df6/onekey_client/cli/ci.py#L10